diff --git a/node_modules/@forge-trade/smart-order-router/build/main/util/addresses.js b/node_modules/@forge-trade/smart-order-router/build/main/util/addresses.js
index 9867c46..c37e37e 100644
--- a/node_modules/@forge-trade/smart-order-router/build/main/util/addresses.js
+++ b/node_modules/@forge-trade/smart-order-router/build/main/util/addresses.js
@@ -19,6 +19,7 @@ const TEVMOS_MULTICALL_ADDRESS = '0x5E9848eBE701d519bcd18fDa07f9432E6fb63dff';
 const EVMOS_V3_CORE_FACTORY_ADDRESSES = '0xf544365e7065966f190155F629cE0182fC68Eaa2';
 const EVMOS_QUOTER_ADDRESSES = '0xacDD67285fFeF73c9C6778019d2fF0A75547048a';
 const EVMOS_MULTICALL_ADDRESS = '0xcF30595B19B299664e8d2CedF41EC8FA859F97b1';
+const OPTIMISM_GOERLI_MULTICALL_ADDRESS = '0xcF30595B19B299664e8d2CedF41EC8FA859F97b1';
 exports.V3_CORE_FACTORY_ADDRESSES = Object.assign(Object.assign({}, constructSameAddressMap(v3_sdk_1.FACTORY_ADDRESS)), { [chains_1.ChainId.CELO]: CELO_V3_CORE_FACTORY_ADDRESSES, [chains_1.ChainId.CELO_ALFAJORES]: CELO_V3_CORE_FACTORY_ADDRESSES, [chains_1.ChainId.ARBITRUM_GOERLI]: ARBITRUM_GOERLI_V3_CORE_FACTORY_ADDRESSES, 
     // TODO: Gnosis + Moonbeam contracts to be deployed
     [chains_1.ChainId.FUJI]: FUJI_V3_CORE_FACTORY_ADDRESSES, [chains_1.ChainId.TEVMOS]: TEVMOS_V3_CORE_FACTORY_ADDRESSES, [chains_1.ChainId.EVMOS]: EVMOS_V3_CORE_FACTORY_ADDRESSES });
@@ -31,7 +32,7 @@ exports.MIXED_ROUTE_QUOTER_V1_ADDRESSES = {
     [chains_1.ChainId.ROPSTEN]: '0x84E44095eeBfEC7793Cd7d5b57B7e401D7f1cA2E',
     [chains_1.ChainId.GÖRLI]: '0xBa60b6e6fF25488308789E6e0A65D838be34194e',
 };
-exports.UNISWAP_MULTICALL_ADDRESSES = Object.assign(Object.assign({}, constructSameAddressMap('0x1F98415757620B543A52E61c46B32eB19261F984')), { [chains_1.ChainId.CELO]: CELO_MULTICALL_ADDRESS, [chains_1.ChainId.CELO_ALFAJORES]: CELO_MULTICALL_ADDRESS, [chains_1.ChainId.ARBITRUM_GOERLI]: ARBITRUM_GOERLI_MULTICALL_ADDRESS, [chains_1.ChainId.FUJI]: FUJI_MULTICALL_ADDRESS, [chains_1.ChainId.TEVMOS]: TEVMOS_MULTICALL_ADDRESS, [chains_1.ChainId.EVMOS]: EVMOS_MULTICALL_ADDRESS });
+exports.UNISWAP_MULTICALL_ADDRESSES = Object.assign(Object.assign({}, constructSameAddressMap('0x1F98415757620B543A52E61c46B32eB19261F984')), { [chains_1.ChainId.CELO]: CELO_MULTICALL_ADDRESS, [chains_1.ChainId.CELO_ALFAJORES]: CELO_MULTICALL_ADDRESS, [chains_1.ChainId.ARBITRUM_GOERLI]: ARBITRUM_GOERLI_MULTICALL_ADDRESS, [chains_1.ChainId.OPTIMISM_GOERLI]: OPTIMISM_GOERLI_MULTICALL_ADDRESS, [chains_1.ChainId.FUJI]: FUJI_MULTICALL_ADDRESS, [chains_1.ChainId.TEVMOS]: TEVMOS_MULTICALL_ADDRESS, [chains_1.ChainId.EVMOS]: EVMOS_MULTICALL_ADDRESS });
 exports.OVM_GASPRICE_ADDRESS = '0x420000000000000000000000000000000000000F';
 exports.ARB_GASINFO_ADDRESS = '0x000000000000000000000000000000000000006C';
 exports.TICK_LENS_ADDRESS = '0xbfd8137f7d1516D3ea5cA83523914859ec47F573';
diff --git a/node_modules/@forge-trade/smart-order-router/build/main/util/chains.d.ts b/node_modules/@forge-trade/smart-order-router/build/main/util/chains.d.ts
index d1c38ab..46817f1 100644
--- a/node_modules/@forge-trade/smart-order-router/build/main/util/chains.d.ts
+++ b/node_modules/@forge-trade/smart-order-router/build/main/util/chains.d.ts
@@ -7,6 +7,7 @@ export declare enum ChainId {
     KOVAN = 42,
     OPTIMISM = 10,
     OPTIMISTIC_KOVAN = 69,
+    OPTIMISTIC_GOERLI = 420,
     ARBITRUM_ONE = 42161,
     ARBITRUM_RINKEBY = 421611,
     ARBITRUM_GOERLI = 421613,
@@ -17,6 +18,7 @@ export declare enum ChainId {
     GNOSIS = 100,
     MOONBEAM = 1284,
     FUJI = 43113,
+    SEPOLIA = 11155111, 
     TEVMOS = 9000,
     EVMOS = 9001
 }
@@ -31,8 +33,10 @@ export declare enum ChainName {
     RINKEBY = "rinkeby",
     GÖRLI = "goerli",
     KOVAN = "kovan",
+    SEPOLIA = "sepolia",
     OPTIMISM = "optimism-mainnet",
     OPTIMISTIC_KOVAN = "optimism-kovan",
+    OPTIMISTIC_GOERLI = "optimism-goerli",
     ARBITRUM_ONE = "arbitrum-mainnet",
     ARBITRUM_RINKEBY = "arbitrum-rinkeby",
     ARBITRUM_GOERLI = "arbitrum-goerli",
diff --git a/node_modules/@forge-trade/smart-order-router/build/main/util/chains.js b/node_modules/@forge-trade/smart-order-router/build/main/util/chains.js
index 785224c..c4c29b3 100644
--- a/node_modules/@forge-trade/smart-order-router/build/main/util/chains.js
+++ b/node_modules/@forge-trade/smart-order-router/build/main/util/chains.js
@@ -7,7 +7,7 @@ var ChainId;
     ChainId[ChainId["MAINNET"] = 1] = "MAINNET";
     ChainId[ChainId["ROPSTEN"] = 3] = "ROPSTEN";
     ChainId[ChainId["RINKEBY"] = 4] = "RINKEBY";
-    ChainId[ChainId["G\u00D6RLI"] = 5] = "G\u00D6RLI";
+    ChainId[ChainId["GOERLI"] = 5] = "GOERLI";
     ChainId[ChainId["KOVAN"] = 42] = "KOVAN";
     ChainId[ChainId["OPTIMISM"] = 10] = "OPTIMISM";
     ChainId[ChainId["OPTIMISTIC_KOVAN"] = 69] = "OPTIMISTIC_KOVAN";
@@ -122,7 +122,7 @@ var ChainName;
     ChainName["MAINNET"] = "mainnet";
     ChainName["ROPSTEN"] = "ropsten";
     ChainName["RINKEBY"] = "rinkeby";
-    ChainName["G\u00D6RLI"] = "goerli";
+    ChainName["GOERLI"] = "goerli";
     ChainName["KOVAN"] = "kovan";
     ChainName["OPTIMISM"] = "optimism-mainnet";
     ChainName["OPTIMISTIC_KOVAN"] = "optimism-kovan";
@@ -137,7 +137,7 @@ var ChainName;
     ChainName["MOONBEAM"] = "moonbeam-mainnet";
     ChainName["FUJI"] = "avalanche-fuji";
     ChainName["TEVMOS"] = "evmos-testnet";
-    ChainName["EVMOS"] = "evmos-maintnet";
+    ChainName["EVMOS"] = "evmos";
 })(ChainName = exports.ChainName || (exports.ChainName = {}));
 var NativeCurrencyName;
 (function (NativeCurrencyName) {
